index.js

```diff
@@ -14,6 +14,7 @@ function Plugin(options) {
   if (this.options.logTime === undefined) {
     this.options.logTime = DEFAULT_LOG_TIME;
   }
+  this.options.append = options.append || false;
```

Comment:
This probably don't need to be an option, extending always should be good enough as people will just constantly overwrite the same bundle keys. There's no drawback that would require people to turn it off.